database/seeds/OtherDataSeeder.php

```diff
@@ -0,0 +1,141 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+use Proto\Models\HashMapItem;
+use Proto\Models\User;
+use Proto\Models\Committee;
+use Proto\Models\Address;
+use Proto\Models\Bank;
+use Proto\Models\Product;
+use Proto\Models\Member;
+use Proto\Models\Activity;
+use Proto\Models\OrderLine;
+use Proto\Models\CommitteeMembership;
+use Proto\Models\ActivityParticipation;
+
+class OtherDataSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+
+        $faker = Faker\Factory::create();
+
+        // List of valid ID's
+        $products = Product::all()->pluck('id')->toArray();
+        $committees = Committee::all()->pluck('id')->toArray();
+
+        // Create users
+        $n = 500;
+        echo "Creating $n users" . PHP_EOL;
+        factory(User::class, $n)->create()->each(function ($user) {
+            /** @var $user User */
+            if (mt_rand(1, 5) > 1) {
+                $user->address()->save(factory(Address::class)->make());
+            }
+            if (mt_rand(1, 5) > 1) {
+                $user->bank()->save(factory(Bank::class)->make());
+            }
+            if (mt_rand(1, 5) > 1) {
+                $user->member()->save(factory(Member::class)->make());
+            }
+        });
+
+        $users = array_merge(array_rand(User::all()->pluck('id')->toArray(), 75), [1]);
+
+        // Create orderlines
+        $n = 10000;
+        echo "Creating $n orderlines" . PHP_EOL;
+
+        $mintime = date('U', strtotime('April 20, 2011'));
+        $maxtime = date('U', strtotime('now'));
+
+        for ($i = 0; $i < $n; $i++) {
+            $p = Product::find($products[array_rand($products)]);
+            $d = date('Y-m-d H:i:s', mt_rand($mintime, $maxtime));
+            $u = mt_rand(1, 3);
+            $data = [
+                'user_id' => $users[array_rand($users)],
+                'product_id' => $p->id,
+                'original_unit_price' => $p->price,
+                'units' => $u,
+                'total_price' => $u * $p->price,
+                'created_at' => $d
+            ];
+
+            if (mt_rand(1, 100) == 1) {
+                // Simulate Pay via Cash
```

Comment:
Payed with card seems more relevant now.

Reply:
Agreed and changed!